XDMA/linux-kernel/xdma/xdma_netdev.c

```diff
@@ -104,7 +105,7 @@ netdev_tx_t xdma_netdev_start_xmit(struct sk_buff *skb,
         /* Check desc count */
         netif_stop_queue(ndev);
         xdma_debug("xdma_netdev_start_xmit(skb->len : %d)\n", skb->len);
-        skb->len = (skb->len < ETH_ZLEN) ? (ETH_ZLEN - skb->len) : 0;
+        skb->len += (skb->len < ETH_ZLEN) ? (ETH_ZLEN - skb->len) : 0;
```

Comment:
최소값 리밋을 거는 거라면 이게 더 깔끔합니다.

```suggestion
skb->len = max(ETH_ZLEN, skb->len)
```